.circleci/config.yml

```diff
@@ -132,8 +132,7 @@ commands:
           command: |
             if [[ $CIRCLE_TAG == android-v* ]]; then
               sed -i -e "s/^VERSION_NAME=.*/VERSION_NAME=${CIRCLE_TAG:9}/" gradle.properties
-            fi
-            if [[ $CIRCLE_TAG == main ]]; then
+            elif [[ $CIRCLE_BRANCH == main ]]; then
```

Comment:
was it actually not working?

Reply:
yep, the CIRCLE_TAG is empty for the main branch, and it didn't enter the sed logic.

It's hard to validate until the PR is merged to main 🙃